clients/cobol-lsp-vscode-extension/src/services/LanguageClientService.ts

```diff
@@ -152,9 +152,35 @@ export class LanguageClientService {
     }, 250);
   }
 
-  public async start() {
+  public async start(context: vscode.ExtensionContext) {
     const languageClient = this.getLanguageClient();
-    await languageClient.start();
+    try {
+      await languageClient.start();
+    } catch {
+      if (SettingsService.serverRuntime() === "NATIVE") {
+        vscode.window.showInformationMessage(
+          "Native Server Runtime was failed to start. Try to use Java Server Runtime",
+        );
+        vscode.commands.executeCommand(
+          "workbench.action.openSettings",
+          `@ext:${context.extension.id}`,
+        );
```

Comment:
I think this should be also triggered after clicking a button.

Also I think it would be better to turn the whole catch block into a function.